client/structs.h

```diff
@@ -119,3 +119,36 @@ typedef struct _TDNF_EVENT_DATA_
     struct _TDNF_EVENT_DATA_ *pNext;
 }TDNF_EVENT_DATA, *PTDNF_EVENT_DATA;
 
+//Metalink Structures.
+typedef struct _TDNF_LIST_
```

Comment:
this name is also very generic, TDNF_LIST.

Reply:
done